### PR TITLE
feat(event-db): NPG-5753 GraphQL schema server and security definitions

### PR DESCRIPTION
**Pull request by stevenj**
Add GraphQL server for admin and security.

### CODE REVIEW

File: schema.graphql
Comment (Mr-Leshiy) [inline schema.graphql:1]: Is it possible to split it into the separate files ?

File: src/event-db/setup/graphql-setup.sql [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    about           text,
+    email           text not null unique check (email ~* '^.+@.+\..+$'),
+    password_hash   text not null,
+    created_at      timestamp default now()
+);
+
+-- Create default ADMIN user.
+
+insert into private.admin_account (first_name, last_name, role, about, email, password_hash) values
+  ('Admin', 'Default', 'cat_admin', 'Default Admin User', 'admin.default@nowhere.io', crypt('CHANGE_ME', gen_salt('bf')));
```
Comment (cong-or) [inline src/event-db/setup/graphql-setup.sql:56]: should there be a larger salt?

Comment (stevenj) [reply src/event-db/setup/graphql-setup.sql:56]: gen_salt generates a salt with the 'bf' type.
See: https://www.postgresql.org/docs/current/pgcrypto.html#PGCRYPTO-CRYPT-ALGORITHMS
its a 128-bit salt.

File: schema.graphql
Comment (stevenj) [reply schema.graphql:1]: Its autogenerated.

**Review by cong-or — APPROVED** [review]
LGTM

**Pull request merged**